lib/rubocop/remote_config.rb

```diff
@@ -26,6 +28,12 @@ def file
       cache_path
     end
 
+    def inherit_from_remote(file, path)
+      new_uri = URI.parse(@uri.to_s)
```

Comment:
How come `@uri` needs to be re-parsed here? If you need a copy, `#dup` is probably a better candidate. 🙂